### DISCUSSION TITLE
AO 0.7 keeps ruining scenery_packs.ini order

### DISCUSSION THREAD

**Discussion by cyrusyn** [kubilus1/autoortho#427] [General]
Just installed 0.7 and started XP12. Loading time improved a lot but the scenery (buildings, roads and other structures and trees) disappeared. Turned out XP keeps putting Z_AO... folders at the very top of the scenery_packs.ini. I fixed it and relaunched XP and the result was same. Restarted my rig a few time and reeditted the ini file and XP keeps doing this. Amy one having the same issue?

**Comment by rquintelarj** [comment]
Run autoortho after XP12 initial loading. When the main menu appears, you click run in autoortho.

**Comment by chezhj** [comment]
It's most likely caused by SAM. If you are using SAM configure it with manual links, not automatic

**Reply by cyrusyn** [reply]
I don't have SAM on my XP12. I will double check in case I installed long time ago.

**Reply by Medeir0s95** [reply]
How can i set this???

**Reply by chezhj** [reply]
Go to Plugins -> SAM -> Airport Operation System. In settings choose manually.
Create links manually.
